src/main/java/recipeio/commands/AddRecipeCommand.java

```diff
@@ -12,12 +12,14 @@ public class AddRecipeCommand {
      *
      * @param recipe The new recipe to be added.
      */
-    public static void execute(Recipe recipe, ArrayList<Recipe> recipes) {
+    public static void execute(Recipe recipe, ArrayList<Recipe> recipes, boolean silentLoad) {
         //this try-catch block can be helpful when there's not enough memory for a new recipe in the list
         try {
             //TaskList<T> add method only throw IndexOutOfBound exception for the overload add(int, T).
             recipes.add(recipe);
-            UI.printAddMessage(recipe, recipes.size());
+            if (!silentLoad) {
+                UI.printAddMessage(recipe, recipes.size());
```

Comment:
Actually, you could consider moving the UI.printMessage from the AddRecipeCommand into the executeCommand method in RecipeList.java, inside the case function.

That way, the printMessage is only called when executeCommand is called, i.e. when the user adds a recipe.

Hopefully that will solve the Text Ui test error you're getting.